### PR TITLE
Ignore comment lines

### PR DESCRIPTION
**Pull request by afontani**
This pull request is an effort to make the housing characteristics more machine-readable. The created by line is causing issues when reading the files programmatically. The resolution is that we have ResStock ignore comment lines. The comment lines begin with the "#" symbol. 

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [x] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [x] `project_testing` runs without any failures
- [x] No unexpected regression test changes
- [x] All tests are passing (green) on circleci
- [x] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst [hunk 1]
```diff
@@ -7,4 +7,5 @@ Guidelines
 =============
 
 1. All lines have the line ending characters '\\r\\n' (i.e., "crlf" or "carriage return, line feed").
-2. All non-dependency columns have format '%.6f'.
\ No newline at end of file
+2. All non-dependency columns have format '%.6f'.
+3. Comment lines at the end of a housing characteristics file, indicated by the "#" symbol, can be added.
```
Comment (shorowit) [inline docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst]: The implementation appears to handle comment lines anywhere in the file, not just at the end of the file, right?

Comment (afontani) [reply docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst]: Good point. I didn't include the necessary lines of code for header comments, and comments inserted in the middle of the tsv.
- Header comments: looks like I need to update get_file_data(...)
- Comments inserted in the middle of the tsv: looks like I need to update get_option_name_from_sample_number(...)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,11 @@ Features
 - Relocate the data folder, along with tsv makers, to a separate private repository ([#401](https://github.com/NREL/OpenStudio-BuildStock/pull/401))
 - Update the single-family detached and multifamily projects with more up-to-date lighting stock distributions ([#392](https://github.com/NREL/OpenStudio-BuildStock/pull/392))
 - Update Insulation Finished Attic tsv with more options for insulation levels ([#395](https://github.com/NREL/OpenStudio-BuildStock/pull/395))
+- Add ability to ignore comment lines with the "#" symbol ([#408](https://github.com/NREL/OpenStudio-BuildStock/pull/408))
 
 Fixes
 - Refactor the tsv maker classes to accommodate more data sources ([#392](https://github.com/NREL/OpenStudio-BuildStock/pull/392))
+- Revert plug load schedule to RBSA ([#355](https://github.com/NREL/OpenStudio-BuildStock/pull/355))
```
Comment (joseph-robertson) [inline CHANGELOG.md]: You might mention that this is for the National Average option in the options lookup file.

**Comment by afontani** [comment]
@joseph-robertson Good point. I updated the CHANGLOG.md.

**Pull request merged**